### PR TITLE
Feat/local notifications

### PR DESCRIPTION
**Pull request by Luftzig**

### CODE REVIEW

File: android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeFirebaseMessagingService.java [hunk 1]
```diff
@@ -121,4 +123,13 @@ private void runBackgroundTask(Context context, Bundle bundle) {
             Log.w(TAG, "No task name");
         }
     }
+
```
Comment (tobiasr) [inline android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeFirebaseMessagingService.java:126]: Below seems to a whitespace change or something like that? Maybe it should be excluded from the PR?

File: android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeNotificationHubModule.java [hunk 2]
```diff
@@ -377,6 +386,54 @@ public void isNotificationEnabledOnOSLevel(Promise promise) {
         promise.resolve(areNotificationsEnabled);
     }
 
+    @ReactMethod
+    public void scheduleLocalNotification(ReadableMap map, String timeMsString, Promise promise) {
```
Comment (tobiasr) [inline android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeNotificationHubModule.java:390]: This is the only "new" feature right? Besides `RCTConvert` for IOS. The rest of the edits are more refactorings. Correct?

**Review by tobiasr — APPROVED** [review]
I think this looks good. Might be better to just merge it than to fix "details"

File: android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeNotificationHubModule.java (hunk 2 above)
Comment (Luftzig) [reply android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeNotificationHubModule.java:390]: Correct

**Pull request merged**